test/http_client_test.exs

```diff
@@ -490,20 +502,22 @@ defmodule Closex.HTTPClientTest do
     end
   end
   describe "get_users/0" do
+    @tag :this
```

Comment:
🔫

Reply:
^^ I hate how on macs this renders as a water pistol :/